src/BooleanSlider/widget/BooleanSlider.js

```diff
@@ -117,8 +117,8 @@ define([
             if (this._contextObj !== null) {
                 dojoStyle.set(this.domNode, "display", "block");
 
-                dojoProp.set(this.trueNode, "textContent", this.trueValue);
-                dojoProp.set(this.falseNode, "textContent", this.falseValue);
```

Comment:
Because when running the deprecation widget I get the result below. According to this document the mxui.dom.textContent should be replaced with mxui.dom.text. 
I am trying to clean all the widget we use, so that when running the mx-check-deprecation I have no errors left :)  

BooleanSlider.mpk	1.4.0	BooleanSlider/widget/BooleanSlider.js	120	44	mxui.dom.textContent	mxui.dom.text	                dojoProp.set(this.trueNode, "textContent", this.trueValue);
BooleanSlider.mpk	1.4.0	BooleanSlider/widget/BooleanSlider.js	121	45	mxui.dom.textContent	mxui.dom.text	                dojoProp.set(this.falseNode, "textContent", this.falseValue);